cmd/vc-rest/startcmd/start.go

```diff
@@ -125,7 +125,7 @@ func createStartCmd(srv server) *cobra.Command {
 			}
 
 			universalResolverURL, err := cmdutils.GetUserSetVar(cmd, universalResolverURLFlagName,
-				universalResolverURLEnvKey, true)
+				universalResolverURLEnvKey, false)
```

Comment:
I noticed that this was set to optional, but it seems to be required since I couldn't go through the REST flow without having a universal resolver running, so I set it to required (isOptional=false). @fqutishat  @sandrask Let me know if you have any concerns.

Reply:
keep it optional

Reply:
I can't reproduce that original issue I was having, so I'll make it optional again

Reply:
Done